core/tests/protractor_desktop/preferences.js

```diff
@@ -26,6 +26,31 @@ describe('Preferences', function() {
     preferencesPage = new PreferencesPage.PreferencesPage();
   });
 
+  it('should upload a profile photo', function() {
+    users.createUser('eve@preferences.com', 'evePreferences');
+    users.login('eve@preferences.com');
+    preferencesPage.get();
+    expect(preferencesPage.getProfilePhotoSource())
+      .not
+      .toEqual(
+        preferencesPage.submitProfilePhoto('../data/img.png')
+          .then(function() {
+            return preferencesPage.getProfilePhotoSource();
+          })
+      );
+  });
+
+  it('shows an error if uploaded photo is too large', function() {
```

Comment:
`should show and error...`

Always begin these tests with should or should not.

Reply:
Fixed in 538c9e0